libsupport/include/katana/Result.h

```diff
@@ -629,7 +629,7 @@ CheckedExpressionToValue(arrow::Status&&) {
     if (::katana::internal::CheckedExpressionFailed(result_name)) {            \
       return ::katana::internal::CheckedExpressionToError(result_name)         \
           .WithContext(__VA_ARGS__)                                            \
-          .WithContext(FMT_STRING("({}:{})"), __FILE__, __LINE__);             \
+          .WithContext(FMT_STRING("\n({}:{})"), __FILE__, __LINE__);           \
```

Comment:
I tried that, does not seem better:
"({}:{})\n"

```
g/forked/katana-enterprise/libquery/src/OpGraph.cpp:784)
: backtrace: (/home/cpchung/kg/forked/katana-enterprise/libquery/src/error-checker-passes/SymbolResolutionChecker.cpp:224)
: backtrace: (/home/cpchung/kg/forked/katana-enterprise/libquery/src/error-checker-passes/SymbolResolutionChecker.cpp:418)
: backtrace: (/home/cpchung/kg/forked/katana-enterprise/libquery/src/error-checker-passes/SymbolResolutionChecker.cpp:418)
: backtrace: parameter name endDate is undefined (SymbolResolutionChecker.cpp:355)
TCK = Undefined Parameter Error
Katana = ParameterError:UndefinedParameter
```

Reply:
It seems more correct to me as now each line represents a frame and the line number is at the end of the corresponding frame rather than on the following line.

It looks weird because many of the intervening frames have not next associated with them and the initial `:` comes from the fact the message is intended to be rendered on one line.

Reply:
I kinda like the idea of splitting up the trace into multiple lines. But it will require more than changing this I think. Since we should format it more carefully. I think one line might be the best we can do if we handle line information as strings instead of actually having a backtrace data structure like most exception systems.